infra/modules/aws-cognito-identity-cli/main.tf

```diff
@@ -0,0 +1,21 @@
+resource "null_resource" "identity-command" {
+  for_each = var.login-ids
+
+  triggers = {
+    always_run = "${timestamp()}"
+  }
+  provisioner "local-exec" {
+    command = "bash ${path.module}/addIdentity.sh ${var.identity_pool_id} ${each.value} ${var.aws_region} ${var.aws_role} ${each.key} ${path.module}"
+  }
+}
+
+data "local_file" "identity-result" {
+  for_each = var.login-ids
+
+  filename   = "${path.module}/cognito_identity_${each.key}.json"
+  depends_on = [null_resource.identity-command]
+}
```

Comment:
so yeah, my preference is a single `data` block that invokes the script, parses STDOUT as JSON and gets the value from there - rather than writing a file. I think 1) will be more concise/clear in terraform code, 2) will avoid extraneous file persisted to disk. I think it's similar to what we did in the certificate case previously